rmw_implementation_python/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+cmake_minimum_required(VERSION 3.5)
+
+project(rmw_implementation_python)
+
+find_package(ament_cmake REQUIRED)
+find_package(ament_cmake_python REQUIRED)
+
+ament_python_install_package(${PROJECT_NAME})
```

Comment:
This could probably be made into a pure Python package with a `setup.py` rather than a `CMakeLists.txt`, but this also works as-is. I just wanted to make sure you were aware it was possible.

Reply:
This should really not be a Python package. So :-1: from me to merge this as-is.

Reply:
> This should really not be a Python package.

It isn't a Python package right now? Did you mean it should really be a Python package?

Reply:
That's indeed what I meant.